static/app/components/tooltip.tsx

```diff
@@ -79,6 +79,11 @@ type Props = DefaultProps & {
    */
   forceShow?: boolean;
 
+  /**
+   * Only display the tooltip if the content overflows
+   */
+  showOnEllipsis?: boolean;
```

Comment:
imo `overflow` describes the behavior better than `ellipsis`, the latter describes how it's presented instead of what is happening

Reply:
I agree, I've changed the variable names to use `overflow`, thanks!